app/controllers/network_services_controller.go

```diff
@@ -1367,6 +1395,36 @@ func generateIpPortId(ip, protocol, port string) string {
 	return ip + "-" + protocol + "-" + port
 }
 
+// returns all IP addresses found on any network address in the system, excluding dummy and docker interfaces
+func getAllLocalIPs() ([]netlink.Addr, error) {
+	links, err := netlink.LinkList()
+	if err != nil {
+		return nil, errors.New("Could not load list of net interfaces: " + err.Error())
+	}
+
+	addrs := make([]netlink.Addr, 0)
+	for _, link := range links {
+
+		// do not include IPs for any interface that calls itself "dummy"
+		// or any of the docker# interfaces
+		if strings.Contains(link.Attrs().Name, "dummy") ||
```

Comment:
thanks @dl00 for the patch. Functionally this should achive expected node port behaviour. 

Is is possible for you check the link attributes to exclude interfaces of bridge type and NOARP interfaces, to get the list of IP address?

Reply:
I could, I just dont want any interfaces to be missed that should be included. For example, my infrastructure uses a `tap0` interface which might qualify as a bridge type, and I would not want it excluded.

NOARP I am not familiar with, but if it makes sense it would probobly be trivial enough check to add.